cypress-tests/cypress/e2e/spec/Payment/00031-UCSComprehensiveTest.cy.js

```diff
@@ -0,0 +1,583 @@
+import * as fixtures from "../../../fixtures/imports";
+import State from "../../../utils/State";
+import getConnectorDetails from "../../configs/Payment/Utils";
+import { cardCreditEnabled } from "../../configs/PaymentMethodList/Commons";
+
+let globalState;
+
+describe("UCS Comprehensive Test", () => {
+  before("Initialize and Setup", function () {
+    const UCS_SUPPORTED_CONNECTORS = ["authorizedotnet"];
+    cy.task("getGlobalState").then((state) => {
+      globalState = new State(state);
+      const connectorId = Cypress.env("CYPRESS_CONNECTOR");
+      if (!UCS_SUPPORTED_CONNECTORS.includes(connectorId)) {
+        cy.log(`Skipping UCS tests - connector not supported: ${connectorId}`);
+        this.skip();
+      }
```

Comment:
can you use inclusion list from `cypress-tests/cypress/e2e/configs/Payment/Utils.js` instead?